soundboard/src/main/java/editor/model/EditableGroup.java

```diff
@@ -60,6 +60,24 @@ public boolean isMarkedForRemoval(Playlist playlist) {
         return markedForRemoval.contains(playlist);
     }
 
+    public boolean isGroupNameEdited() {
+        return !updatedName.equals(group.getName());
+    }
+
+    /**
+     * @return true if this Group has been edited in any one of the following ways:
+     * <li>Name edited</li>
+     * <li>Playlist added</li>
+     * <li>Playlist removed</li>
+     */
+    public boolean hasChanges(){
+        return isGroupNameEdited() || !recentlyAdded.isEmpty() || !markedForRemoval.isEmpty();
+    }
```

Comment:
I actually like this, we should definitely be able to query the model for anything data related, and then the controller can decide what to do with that information.